client/gutenberg/extensions/slideshow/edit.js

```diff
@@ -72,6 +77,12 @@ class SlideshowEdit extends Component {
 				setAttributes( {
 					images: [ ...imagesNormalized, ...currentImages ],
 				} );
+				const anyBlobs = imagesNormalized.every( image => {
+					return isBlobURL( image.url );
+				} );
```

Comment:
Super-nitty nit, could shorten to `image => isBlobURL( image.url )` (to implicitly use the return value of the fat-arrow's RHS).